### PR TITLE
Add full support for Glacier engine

### PR DESCRIPTION
**Pull request by thecatontheceiling**
### SteamDB app page links to a few games using this

https://steamdb.info/app/1659040/
https://steamdb.info/app/863550/
https://steamdb.info/app/236870/
https://steamdb.info/app/203140/
https://steamdb.info/app/205930/
https://steamdb.info/app/247430/
https://steamdb.info/app/6860/

### CODE REVIEW

**Comment by SteamDB-Tracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.Glacier**

**7** apps still match.

These apps now match: **(15)** [6850](https://steamdb.info/app/6850/depots/) [6860](https://steamdb.info/app/6860/depots/) [6900](https://steamdb.info/app/6900/depots/) [6950](https://steamdb.info/app/6950/depots/) [8080](https://steamdb.info/app/8080/depots/) [28000](https://steamdb.info/app/28000/depots/) [28020](https://steamdb.info/app/28020/depots/) [32900](https://steamdb.info/app/32900/depots/) [35000](https://steamdb.info/app/35000/depots/) [35050](https://steamdb.info/app/35050/depots/) [203140](https://steamdb.info/app/203140/depots/) [247430](https://steamdb.info/app/247430/depots/) [299050](https://steamdb.info/app/299050/depots/) [610860](https://steamdb.info/app/610860/depots/) [1347780](https://steamdb.info/app/1347780/depots/)

Please confirm that all these changed apps are correct.

File: rules.ini [hunk 1]
```diff
@@ -101,7 +101,10 @@ GDevelop[] = (?:^|/)LICENSE\.GDevelop\.txt$
 GDevelop[] = (?:^|/)gdjs-evtsext__
 GDevelop[] = (?:^|/)GDCpp\.dll$
 Gideros = gideros
-Glacier = \.rpkg$
+Glacier[] = \.rpkg$
+Glacier[] = \.pc_packagelist$
+Glacier[] = ^(?:Scenes)\/(?:.+\/)?[^\/]+\.(?:WHD)$
+Glacier[] = ^Music\/Low\/[^\/]+\/[^\/]+\.sty$
```
Comment (Lifeismana) [inline rules.ini:107]: these 2 regex are unreadable, this won't be merged in this current state
And `.+` / any usage of + is something that must be avoided

Comment (thecatontheceiling) [reply rules.ini:107]: Will fix when I get the chance, sorry 

I know it's a mess lol

Comment (thecatontheceiling) [reply rules.ini:107]: Fixed

Comment (thecatontheceiling) [reply rules.ini]: Okay this is extremely annoying, SteamDB doesnt support wildcards?

Comment (Lifeismana) [reply rules.ini]: using .* to replace .+ is not a fix, it's the same things but also allowing 0 caracters

**Comment by SteamDB-Tracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.Glacier**

**7** apps still match.

These apps now match: **(12)** [6850](https://steamdb.info/app/6850/depots/) [6860](https://steamdb.info/app/6860/depots/) [6900](https://steamdb.info/app/6900/depots/) [6950](https://steamdb.info/app/6950/depots/) [8080](https://steamdb.info/app/8080/depots/) [28000](https://steamdb.info/app/28000/depots/) [28020](https://steamdb.info/app/28020/depots/) [35000](https://steamdb.info/app/35000/depots/) [35050](https://steamdb.info/app/35050/depots/) [203140](https://steamdb.info/app/203140/depots/) [247430](https://steamdb.info/app/247430/depots/) [1347780](https://steamdb.info/app/1347780/depots/)

Please confirm that all these changed apps are correct.

**Comment by thecatontheceiling** [comment]
I have regex so fucking much that I haven't bothered to write something actually usable in 3 months. I'll just close and let someone more competent do it lol

**Pull request closed without merging**